extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -39,17 +39,18 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
     onGeoJsonParsed: OnGeoJsonParsed
   ) : this(builder) {
     rawGeoJson?.let {
-      ignoreParsedGeoJsonRegistry[sourceId] = false
+      val dataKey = sourceId + it.hashCode()
```

Comment:
why can't it be simply hash code?

Reply:
If set the same data to two different sources, they will affect each other.